app/pubspec.yaml

```diff
@@ -61,7 +61,7 @@ dependencies:
       ref: main
 
   flutter_easyloading: ^3.0.5
-  appflowy_editor: ^4.0.0
+  appflowy_editor: ^5.1.0
```

Comment:
also to upgrade dependencies before for improvements!

Reply:
yay \o/